common/amount.h

```diff
@@ -148,6 +148,12 @@ bool amount_msat_eq_sat(struct amount_msat msat, struct amount_sat sat);
 /* a / b */
 double amount_msat_ratio(struct amount_msat a, struct amount_msat b);
 
+/* floor(a/b) */
+u64 amount_msat_ratio_floor(struct amount_msat a, struct amount_msat b);
```

Comment:
u64?  I'd expect these to return double?

Reply:
Floor(x): the biggest integer that is not greater than x.
It is integer anyways.